tests/integration/test_dynamodb.py

```diff
@@ -41,53 +36,55 @@ def transcribe_snapshot_transformer(snapshot):
 
 
 class TestDynamoDB:
-    @pytest.mark.only_localstack
-    def test_non_ascii_chars(self, aws_client):
-        resources.create_dynamodb_table(TEST_DDB_TABLE_NAME, partition_key=PARTITION_KEY)
+    @pytest.fixture
+    def ddb_test_table(self, aws_client) -> str:
+        """
+        This fixture returns a DynamoDB table for testing.
+        """
+        table_name = "ddb-test-table"
```

Comment:
```suggestion
        table_name = "ddb-test-table"
```

Would suggest making this unique to avoid any potential issues/conflicts and allow for potential parallelization in the future

Reply:
Fixed in e274007f63a9db0208ff40f1a58aac91be4b7a83